BossMod/Framework/WorldStateGameSync.cs

```diff
@@ -588,6 +588,18 @@ private unsafe void UpdateClient()
         var levels = uiState->PlayerState.ClassJobLevels;
         if (!MemoryExtensions.SequenceEqual(_ws.Client.ClassJobLevels.AsSpan(), levels))
             _ws.Execute(new ClientState.OpClassJobLevelsChange(levels.ToArray()));
+
+        Span<uint> blueSpells = stackalloc uint[24];
+        blueSpells.Clear();
+        for (var i = 0; i < 24; i++)
+            blueSpells[i] = actionManager->GetActiveBlueMageActionInSlot(i);
```

Comment:
Can just compare actionmanager's BlueMageActions array directly, no need to call some function 24 times